src/scss/_collaboration-pricing.scss

```diff
@@ -18,18 +18,34 @@
   }
 
   .collaboration-plans {
-    display: flex;
     margin: 50px -10px 10px;
-    overflow-x: auto;
     padding: 20px 10px;
+
+    @include respond-to(large-desktop) {
```

Comment:
Nie podoba mi się powielanie tych samych właściwości w różnych `respond-to`, nie mam pewności, ale chyba można to zrobić w taki sposób:
```
@include respond-to(large-desktop),
@include respond-to(desktop) {
 ...
}
```
To by poprawiło czytelność. Ta sama sytuacja w lini 42. i 81.

Reply:
i 168. i 179. :D